### PR TITLE
MAINT: update codeowners.

### PR DESCRIPTION
**Pull request by tupui**
Please update as wished 😃 

(Reminder that only people listed in the maintainer team can be listed in this file, it's a GH thing.)

### CODE REVIEW

**Review by tylerjereddy — COMMENTED** [review]
LGTM, and the GitHub has a green validation check visible in the diff. I guess we could wait another day or two for the other folks you've added here to confirm if we really want, but otherwise you could probably merge?

**Review by mdhaber — APPROVED** [review]

File: .github/CODEOWNERS
Comment (andyfaff) [inline .github/CODEOWNERS:59]: .travis.yml can be removed I think

File: .github/CODEOWNERS
Comment (tupui) [inline .github/CODEOWNERS:57]:
```suggestion
```

**Comment by tupui** [comment]
Thanks all. There is no rush so I will leave this open a bit in case.

**Comment by rgommers** [comment]
@tupui thanks for updating this. Would you mind adding me for all `meson.build` files?

**Comment by tupui** [comment]
> @tupui thanks for updating this. Would you mind adding me for all `meson.build` files?

Done. I did `meson*` to cover the option file. If I understood the syntax correctly that should work.

Feel free to adjust in general 😃

**Comment by h-vetinari** [comment]
While thinking about #18298, I was wondering who the codeowner for `scipy/integrate` is, and it seems this entry is missing.

Other folders that are missing AFAICT:
```
scipy/_build_utils
scipy/_lib
scipy/cluster
scipy/constants
scipy/datasets
scipy/io
scipy/misc
scipy/ndimage
scipy/odr
```

**Comment by rgommers** [comment]
Those don't have a dedicated owner. For some, like `_build_utils`, it may be me as the default/fallback - but I don't want to be listed, because I don't want to get pinged for each PR that touches them. CODEOWNERS is not intended to be a comprehensive listing of all maintainers, instead it is for folks who want to actively opt into getting pings for each PR that touches that file/directory.

**Comment by h-vetinari** [comment]
> CODEOWNERS is not intended to be a comprehensive listing of all maintainers, instead it is for folks who want to actively opt into getting pings for each PR that touches that file/directory.

Is there something in between "gets pinged for every PR that touches a given path" and "can be asked if there are fundamental questions/decisions about that module"? If not, would it make sense to introduce something like that (even informally)?

**Comment by rgommers** [comment]
There isn't really an in-between. We had the kind of list you're interested in here, it was `doc/MAINTAINERS.rst`. It was well out of date more often than not though, so we decided to get rid of it. I'm not too interested in reintroducing it, because it's a bunch of churn and it's never clear where some lines are (when do you remove some folks when they're inactive for a while, etc.).

**Comment by h-vetinari** [comment]
> We had the kind of list you're interested in here, it was `doc/MAINTAINERS.rst`. It was well out of date more often than not though, so we decided to get rid of it.

Fine by me, thanks for the context. Realistically, that means any fundamental questions in such modules fall to you or @larsoner, seeing that Pauli isn't really that active anymore. I'm fine with that too, but wanted to at least ask if there's a more fine-grained fallback.

**Comment by tylerjereddy** [comment]
The merge conflict was because `main` already had the Azure config deleted, so that was trivial to resolve. I squash-merged this in--it has approvals, and release branching is coming in ~10 days or so (optimistically, at least...).

The GitHub UI also shows a green check in the diff for a valid codeowners file, so flushing the CI is pointless.

**Pull request merged**
